src/isotools/_transcriptome_io.py

```diff
@@ -384,9 +398,13 @@ def add_sample_from_bam(self, fn, sample_name=None, barcode_file=None, fuzzy_jun
                                         min_align_fraction)
                         min_align_fraction = 0
 
+                    # handling non-chimeric alignments
+                    # if the alignment is neither chimeric nor low-quality, update the count of non-chimeric reads (total_nc_reads_chr dictionary)
```

Comment:
Instead of explaining the variable name I would change it to `total_nonchimeric_reads_chr`